habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hab3_planner.yaml

```diff
@@ -15,4 +15,9 @@ hierarchical_policy:
     allow_other_place: False
     max_search_depth: 8
     is_reactive: False
+    # This is the number of non-overlapping steps to sub-sample from each
+    # planned sequence (so the agent only does a part of the overall task).
+    plan_split_len: 4
```

Comment:
Does this split the plan? Shouldn't we split the goals instead?

Reply:
Yeah good point. That's a better way to do it. Let me fix the code to have it like that instead.